### PR TITLE
Source List: Create separate columns for source title and siglum

### LINKED ISSUE #1216: Include source title in table on source list page
A request from Debra. To improve findability of sources on the source list page, can we add source title as one of the fields in the table of sources.

Screenshot of current table:

image

> If we can have "Source," "Summary," "Date/Origin," "Siglum," "Image Link," "Chants/Melodies" that would make great view of our contents.

### PR DESCRIPTION
**Pull request by jacobdgm**
This PR fixes #1216.

- There is a new "Siglum" column on the far left, and changes the "Source" column to display the source's title (rather than its siglum).
- Sources' titles are now truncated at 50 characters
- Sources' descriptions are now truncated at 120 characters (rather than 140).
- Users can now get full titles and longer descriptions by mousing over the relevant cell.
- The Chants/Melodies `<th>` is now split onto two lines, making that column narrower.

Screenshots of the page with a wide window and a narrower one:
Screenshot 2023-12-15 at 3 43 20 PM
Screenshot 2023-12-15 at 3 55 02 PM

### CODE REVIEW

**Comment by jacobdgm** [comment]
We sent Debra these screenshots, and are waiting for feedback/approval. Marking this PR as a draft for now - if/once we receive Debra's approval, we can mark this as ready-for-review.

**Comment by jacobdgm** [comment]
Debra likes it! Marking this as ready-for-review!

**Review by dchiller — APPROVED** [review]

**Pull request merged**
